integrationtests/nftIssueCreateBurn_test.go

```diff
@@ -0,0 +1,156 @@
+//go:build integrationtests
+
+package integrationtests
+
+import (
+	"encoding/json"
+	"math/big"
+	"testing"
+
+	indexerdata "github.com/ElrondNetwork/elastic-indexer-go"
+	"github.com/ElrondNetwork/elastic-indexer-go/mock"
+	"github.com/ElrondNetwork/elrond-go-core/core"
+	coreData "github.com/ElrondNetwork/elrond-go-core/data"
+	dataBlock "github.com/ElrondNetwork/elrond-go-core/data/block"
+	"github.com/ElrondNetwork/elrond-go-core/data/esdt"
+	"github.com/ElrondNetwork/elrond-go-core/data/indexer"
+	"github.com/ElrondNetwork/elrond-go-core/data/transaction"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	expectedNonFungible = `{"name":"NON-token","ticker":"NON","token":"NON-abcd","issuer":"61646472","currentOwner":"61646472","type":"NonFungibleESDT","timestamp":5040,"ownersHistory":[{"address":"61646472","timestamp":5040}]}`
```

Comment:
we should decide either to abbreviate or not :)

Reply:
renamed